src/components/Header/ListenerWidth.jsx

```diff
@@ -0,0 +1,18 @@
+import { useState, useEffect } from 'react'
+
+function getWindowDimensions() {
+    const width = window.innerWidth
+    return width
+}
+
+export default function useScreenWidth() {
+    const [windowDimensions, setWindowDimensions] = useState(getWindowDimensions());
+    useEffect(() => {
+        function handleResize() {
+            setWindowDimensions(getWindowDimensions());
+        }
+        window.addEventListener('resize', handleResize);
+        return () => window.removeEventListener('resize', handleResize);
+    }, []);
+    return windowDimensions;
+}
```

Comment:
If you will succeed with css pls remove this hook)

Reply:
Yes)) I will remove this hook and use CSS